packages/utils/src/initialize-cw-test-env/index.ts

```diff
@@ -0,0 +1,162 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+// Keep dotenv import and config before everything else
+import { MetriportMedicalApi } from "@metriport/api-sdk";
+import { patients } from "./patients";
+import axios from "axios";
+import * as AWS from "aws-sdk";
+import { getEnvVarOrFail } from "../shared/env";
+
+export type Doc = {
+  description: string;
+  fileName: string;
+};
+
+const AXIOS_TIMEOUT_MILLIS = 20_000;
+
+/**
+ * Initializes cw test env in staging
+ */
+
+const stagingTestAccountAPIKey = getEnvVarOrFail("COMMONWELL_ORG_PRIVATE_KEY");
+const facilityId = getEnvVarOrFail("FACILITY_ID");
+
+const orgId = getEnvVarOrFail("DOCUMENT_CONTRIBUTION_ORGANIZATION_ID");
+const orgName = getEnvVarOrFail("DOCUMENT_CONTRIBUTION_ORGANIZATION_NAME");
+
+const docUrl = getEnvVarOrFail("DOCUMENT_CONTRIBUTION_URL");
+const stagingApiUrl = getEnvVarOrFail("METRIPORT_API_URL");
+const bucketName = getEnvVarOrFail("S3_BUCKET_NAME");
+const seedBucketName = getEnvVarOrFail("S3_SEED_BUCKET_NAME");
+const region = getEnvVarOrFail("REGION");
+
+const s3 = new AWS.S3({ signatureVersion: "v4", region });
+
+async function main() {
+  const metriportAPI = new MetriportMedicalApi(stagingTestAccountAPIKey, {
+    baseAddress: stagingApiUrl,
+  });
+
+  const facilityPatients = await metriportAPI.listPatients(facilityId);
+
+  for (const patient of patients) {
+    let currentPatient = facilityPatients.find(p => p.firstName === patient.patient.firstName);
+
+    if (!currentPatient) {
+      currentPatient = await metriportAPI.createPatient(patient.patient, facilityId);
+      console.log(`Created patient ${JSON.stringify(currentPatient, null, 2)}`);
+    }
+
+    let index = 0;
+
+    if (currentPatient) {
+      for (const doc of patient.docs) {
+        await addDocumentToS3AndToFHIRServer(currentPatient?.id ?? "", doc, index);
+
+        index += 1;
+      }
+    }
+  }
+}
+
+async function addDocumentToS3AndToFHIRServer(
+  patientId: string,
+  doc: Doc,
+  docIndex: number
+): Promise<void> {
+  const fhirApi = axios.create({
+    timeout: AXIOS_TIMEOUT_MILLIS,
+    baseURL: `${stagingApiUrl}/fhir/R4`,
+    headers: {
+      "x-api-key": stagingTestAccountAPIKey,
+    },
+  });
+
+  const obj = await s3.getObject({ Bucket: seedBucketName, Key: doc.fileName }).promise();
+
+  const uploaded = await s3
+    .upload({
+      Bucket: bucketName,
+      Key: doc.fileName,
+      Body: obj.Body,
+      ContentType: obj.ContentType,
+    })
+    .promise();
+
+  console.log(`Uploaded, file info: ${JSON.stringify(uploaded)}`);
```

Comment:
Why do we `getObject` then we `upload` it again?

nit: on the message, we say `Upload` but it might already be there.

Reply:
We are getting the object from one bucket and adding it to another bucket

Reply:
oh, the bucket is diff! I looked at the key... 😄